### PR TITLE
Basic Functionality

### LINKED ISSUE #1: Create Ui, Parser, Command, Component classes
Similar to iP, basic functionality of the project requires:

* `Ui` class to display messages
* `Parser` class to parse user input
* `Command` class to execute commands parsed

as well as a separate class unique to this project

* `Component` to represent the electrical components

### PR DESCRIPTION
**Pull request by hughjazzman**
Adds
* `Ui`
* `Parser`
* `Command`
* `Component`

Closes #1

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Duke
 
-{Give product intro here}
+CLIrcuit Assistant is a desktop app to implement and solve simple circuits, optimized for use via the Command Line Interface (CLI).
```
Comment (dorianfong98) [inline docs/README.md:3]: Good description.

File: src/main/java/seedu/duke/ui/Ui.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+import java.util.Scanner;
+
+public class Ui {
+    private static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
```
Comment (sevenseasofbri) [inline src/main/java/seedu/duke/ui/Ui.java:11]: Consider removing this.

**Review by sevenseasofbri — APPROVED** [review]

File: src/main/java/seedu/duke/ui/Ui.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
+package seedu.duke.ui;
+
+import java.util.Scanner;
+
+public class Ui {
+    private static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
```
Comment (dorianfong98) [inline src/main/java/seedu/duke/ui/Ui.java:7]: Perhaps it's time for a new ASCII name art? :)

**Review by dorianfong98 — COMMENTED** [review]
LGTM

**Pull request merged**